lib/matplotlib/tests/test_axes.py

```diff
@@ -982,6 +982,23 @@ def test_canonical():
     ax.plot([1, 2, 3])
 
 
+@image_comparison(baseline_images=['arc_angles'])
```

Comment:
add `remove_text=True` and `style='default'` to the decorator to get the new defaults + drop all of the text (which makes the test more stable).